sqlengine/postgres_engine_test.go

```diff
@@ -405,6 +405,45 @@ var _ = Describe("PostgresEngine", func() {
 			})
 		})
 
+		Context("When setting the search path", func() {
+			BeforeEach(func() {
+				connectionString := postgresEngine.URI(address, port, dbname, createdUser, createdPassword)
+				db, err := sql.Open("postgres", connectionString)
+				defer db.Close()
+
+				_, err = db.Exec("CREATE SCHEMA isolated")
+				Expect(err).ToNot(HaveOccurred())
+			})
+
+			AfterEach(func() {
+				connectionString := postgresEngine.URI(address, port, dbname, createdUser, createdPassword)
+				db, err := sql.Open("postgres", connectionString)
+				defer db.Close()
+
+				_, err = db.Exec("DROP SCHEMA isolated CASCADE")
+				Expect(err).ToNot(HaveOccurred())
+			})
+
+			It("creates a user which can only select", func() {
```

Comment:
this test is called "creates a user..." - but I don't see a creation of user in the test. Am I missing something here?

Reply:
Test needs a rename, good spot